### DISCUSSION TITLE
Specify your own cert and key files

### DISCUSSION THREAD

**Discussion by Blackbaud-JasonBodnar** [BrowserSync/browser-sync#2002] [Q&A]
When running browser-sync from the command line (`browser-sync start -s 'demo' --https`) is it possible to specify your own cert and key files so chrome doesn't complain about an invalid certificate?

**Comment by edwinbradford** [comment]
Came across your question whilst researching mine, maybe too late now but can you do this:
````
 browser-sync start --config 'bs-config.js'
````
and then in the `bs-config.js` file:
````
module.exports = {
  "https": {
    "cert": "/path/to/certs/my-cert.pem",
    "key": "/path/to/private/my-cert-key.pem"
  }
};
````
That's what I had to do to get my SSL working in the past.
